### ISSUE TITLE
Uncaught Exception: SQLiteBlobTooBigException

### ISSUE THREAD

**Issue by jmikkelson** [BIDMCDigitalPsychiatry/LAMP-platform#811]
**Describe the bug**
We recently received this error from a study participant "Error message occurring when trying to login and complete mindLAMP daily surveys, Stackoverflow" 
Exception: SQLiteBlobTooBigException. Have you encountered this error before, and do you have any recommendations for resolving it?

**Screenshots**
![6953180634735148619 jpg - Matilda Tonkovic](https://github.com/BIDMCDigitalPsychiatry/LAMP-platform/assets/158356852/09199117-67c9-4ff0-8b89-11e06f04473f)

**Smartphone (please complete the following information):**
 - Device: Motorola E32
 - OS: Android 11

**Comment by ZCOEngineer** [comment]
@jmikkelson We shall check and reply to this

**Comment by sijitg** [comment]
@jmikkelson Does the user see this error screen always while opening the app? If so can we get the complete log since part of the stack trace is not visible in the screenshot.  
Also could you confirm  app version, please check this by tapping App info . 
![appinfo_screen_mindLamp](https://github.com/BIDMCDigitalPsychiatry/LAMP-platform/assets/63729405/b1c5cd8c-a0c8-4fb3-b236-c09352b4e531). Please note that recent updates to the app is in Open testing.
@carlan1 We need staging environment to further troubleshoot this issue.

**Comment by jmikkelson** [comment]
Thank you for this- working on these first two points. We will email credentials for testing environment.

**Comment by michaelmenon** [comment]
@jmikkelson Thank you for the details. We would like to know the mindLamp app version that this user has installed . They can find it by navigating to App info screen. Long press on mindLamp icon on device home and tap on  'info' icon. Please see the attached screenshot.
<img width="220" alt="screenshot_app_info_mindLamp" src="https://github.com/BIDMCDigitalPsychiatry/LAMP-platform/assets/5271064/6fd74ccd-4a3d-4cf3-bcf6-289d8166fa16">
<img width="286" alt="minLamp_Home" src="https://github.com/BIDMCDigitalPsychiatry/LAMP-platform/assets/5271064/32d7b6c7-39b2-4d60-8acc-97736e4457a5">

**Comment by jmikkelson** [comment]
Hi, I am awaiting their response. We can put the issue on hold for now until we have more info.

**Comment by ZCOEngineer** [comment]
Ok @jmikkelson

**Comment by ertjlane** [comment]
The user had installed 2024.7.12.

**Comment by sijitg** [comment]
We are unable to reproduce the issue here we are reviewing code to add possible fix. We shall update on this.

**Comment by ertjlane** [comment]
Thank you, keep us updated!

**Comment by sijitg** [comment]
We have added some fixes based on code review and submitted Open testing version for google review. We shall update once this version is ready for testing.

**Comment by evinkalengaden** [comment]
Hi @ertjlane 

**If you have already installed TEST BUILD that we have provided earlier connected with Zco Server, Please uninstall the build and proceed installing the new build available now in the store.

The android build Version 2024.8.27 is live and ready for open testing [Staging].

Thanks,
Evin K Alengaden

**Comment by ertjlane** [comment]
Hello @evinkalengaden,

To make sure we are thinking about the same thing, you are referring to downloading the app as beta, per the screenshot attached correct?
![Screenshot_20240828-103255](https://github.com/user-attachments/assets/a4425489-b9cd-40c4-bb5a-237f31a67afc)

**Comment by evinkalengaden** [comment]
Hi @ertjlane ,

Yes, The same build [Mind Lamp (Beta)] as you mentioned in the above screenshot. Download it and let me know your comments.

Thanks,
Evin K Alengaden
